components/PlayerCard/index.js

```diff
@@ -0,0 +1,60 @@
+import Image from "next/image";
+import styled from "styled-components";
+export default function PlayerCard({ player }) {
+  return (
+    <StyledCard>
+      <Image src="/img/players/darts-player.jpg" alt={player.name} width="150" height="150" />
+      <h2>{player.name}</h2>
+      <h3>&quot;{player.nickname}&quot;</h3>
+      <div className="player-info">
+        <span className="player-info-label">HT:</span>
+        <span className="player-info-text">{player.hometown}</span>
+      </div>
+      <div className="player-info">
+        <span className="player-info-label">WR:</span>
+        <span className="player-info-text">{player.worldRanking}</span>
+      </div>
+    </StyledCard>
+  )
+}
+
+const StyledCard = styled.div`
```

Comment:
```suggestion
const StyledCard = styled.li`
```

this should be an li element, since it will later be the direct child of a list element

Reply:
this change to an li element is not the best way of my meaning. It should posible to use a card also as a single element and so is a list not best pratice. In compromise of this case i has change the PlayersList component, with two styled components for ul and li